src/app/core/entity-components/entity-utils/view-components/display-entity/display-entity.component.html

```diff
@@ -1,4 +1,7 @@
-<span [ngClass]="{ clickable: !linkDisabled }">
+<div
```

Comment:
I think this was "<span>" for a reason - otherwise it can't be displayed inline properly, which we do at some occasions? Admittedly, I didn't find any UI issue in the PR deployment now, so maybe this is not relevant any more. But I do remember I explicitly changed this to span a while back.

Reply:
I guess the recent style improvements made this not required any more